idr/tests/test_attributes.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+from attributes import get_phenotypes_for_genelist
+from attributes import get_phenotypes_for_gene
+from attributes import get_similar_genes
+from attributes import get_organism_screenids
+from connections import create_http_session, connection
+from images import images_by_phenotype
+
+from config import idr_base_url, organism
+
+
+class TestAttributes():
+
+    @classmethod
+    def setup_class(cls):
+        cls.session = create_http_session(idr_base_url)
+        cls.conn = connection('idr.openmicroscopy.org', 'public', 'public')
+
+    def test_images_by_phenotype(self):
+
+        images = images_by_phenotype(self.conn)
+        assert (isinstance(images, list) is True)
+
+    def test_get_organism_screenids(self):
+
+        organisms = ['Homo sapiens',
+                     'Saccharomyces cerevisiae',
+                     'Schizosaccharomyces pombe',
+                     'Drosophila melanogaster',
+                     'Mus musculus',
+                     'Arabidopsis thaliana']
+        for idx, organism1 in enumerate(organisms):
+            organism_screen_idlist = get_organism_screenids(idr_base_url,
+                                                            self.session,
+                                                            organism1)
+            if idx < 4:
+                assert (organism_screen_idlist == []) is False
```

Comment:
You don't need the `is False/True`. `assert` automatically treats the result as a bool, so you can just assert `statement`/ `not statement`/ `==` /`!=`, and same for the rest of the tests.